From 482790d634780cb8f38c2df189f7baee73f79108 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 22 Nov 2023 00:47:54 +0000
Subject: [PATCH 1/1] src/Setup/Linux: Specify icon in mime info

Otherwise file managers will just use a generic missing icon icon.
---
 src/Setup/Linux/veracrypt.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/Setup/Linux/veracrypt.xml b/src/Setup/Linux/veracrypt.xml
index 4fc73b90..b266c5e3 100644
--- a/src/Setup/Linux/veracrypt.xml
+++ b/src/Setup/Linux/veracrypt.xml
@@ -2,10 +2,12 @@
 <mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
 	<mime-type type="application/x-veracrypt-volume">
 		<comment>VeraCrypt Volume</comment>
+		<icon name="veracrypt"></icon>
 		<glob pattern="*.hc"/>
 	</mime-type>
 	<mime-type type="application/x-truecrypt-volume">
 		<comment>TrueCrypt Volume</comment>
+		<icon name="veracrypt"></icon>
 		<glob pattern="*.tc"/>
 	</mime-type>
 </mime-info>
-- 
2.42.0

